### ISSUE TITLE
Limit response data

### ISSUE THREAD

**Issue by nemozar** [RonasIT/laravel-swagger#20]
API rerutn list with many items. In test use asserts:

```
assertJsonStructure([
                [
                    "fio",
                    "login",
                    "validbefore"
                    ]
                ]
```

in documents generated list with all items. 

1. How i can set limit response data?
2. How i can set response model with custom example value? In annotations not finded how set different response than real api

**Comment by DenTray** [comment]
@nemozar the main idea of this package is to automate the collection of documentation as much as possible, so the response example generating from your test cases. You can use small trick to minimize response: just write test with the minimal response size to the last order (package save only latest response example in the doc).

As a feature request, we may implement some limit config to define the max count of records in the test examples.

**Comment by DenTray** [comment]
@pirs1337 could you please proceed with solution I described above? Let's add the config which will set the response example limit (please note that for the paginated data it should affect on the `data` field).

Also please proceed the exceptions length, in new Laravel versions it looks huge.

**Comment by DenTray** [comment]
![image](https://user-images.githubusercontent.com/9486872/182127741-c217a91b-5e11-4473-9c48-eb4d5bc0562b.png)

@pirs1337 also please investigate the array unexpected displaying

**Comment by pirs1337** [comment]
@DenTray, ready for testing

**Comment by DenTray** [comment]
@NikitaYakovlev could you please test it?

**Comment by DenTray** [comment]
@NikitaYakovlev thanks! seems it swagger viewer issue, it works fine with elements and rapidoc viewer

**Comment by DenTray** [comment]
Released in [2.2 version](https://github.com/RonasIT/laravel-swagger/releases/tag/2.2)
